db/seeds.rb

```diff
@@ -5,33 +5,33 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-    merch_1 = Merchant.create(name: "Shop Here")
-
-    item_1 = Item.create(name:"jumprope", description:"Pink and sparkly.", unit_price:600, merchant_id:"#{merch_1.id}")
-    item_2 = Item.create(name:"hula hoop", description:"Get your groove on", unit_price:700, merchant_id:"#{merch_1.id}")
-    
-    cust_1 = Customer.create(first_name:"Hannah", last_name:"Warner")
-    cust_2 = Customer.create(first_name:"Kimmy", last_name:"Gibbler") 
-    cust_3 = Customer.create(first_name:"Bob", last_name:"Sagget")
-    cust_4 = Customer.create(first_name:"Uncle", last_name:"Dave")
-    cust_5 = Customer.create(first_name:"Uncle", last_name:"Jessie")
-    cust_6 = Customer.create(first_name:"DJ", last_name:"Tanner")
-
-    invoice_1 = Invoice.create(customer_id:"#{cust_1.id}", status:1)
-    invoice_2 = Invoice.create(customer_id:"#{cust_2.id}", status:1)
-    invoice_3 = Invoice.create(customer_id:"#{cust_3.id}", status:1)
-    invoice_4 = Invoice.create(customer_id:"#{cust_4.id}", status:1)
-    invoice_4 = Invoice.create(customer_id:"#{cust_5.id}", status:1)
-    invoice_5 = Invoice.create(customer_id:"#{cust_6.id}", status:1)
-    
-    invoice_item_1 = InvoiceItem.create(invoice_id:"#{invoice_1.id}", item_id:"#{item_1.id}", status: 1, quantity:1, unit_price:600)
-    invoice_item_2 = InvoiceItem.create(invoice_id:"#{invoice_2.id}", item_id:"#{item_2.id}", status: 1, quantity:1, unit_price:700)
-    invoice_item_3 = InvoiceItem.create(invoice_id:"#{invoice_3.id}", item_id:"#{item_2.id}", status: 2, quantity:1, unit_price:700)
-    invoice_item_4 = InvoiceItem.create!(invoice_id:"#{invoice_4.id}", item_id:"#{item_1.id}", status: 2, quantity:1, unit_price:600)
-    invoice_item_5 = InvoiceItem.create!(invoice_id:"#{invoice_5.id}", item_id:"#{item_1.id}", status: 2, quantity:1, unit_price:600)
-    
-    transaction_1 = Transaction.create(invoice_id:"#{invoice_1.id}")
-    transaction_2 = Transaction.create(invoice_id:"#{invoice_2.id}")
-    transaction_3 = Transaction.create(invoice_id:"#{invoice_3.id}")
-    transaction_4 = Transaction.create(invoice_id:"#{invoice_4.id}")
-    transaction_5 = Transaction.create(invoice_id:"#{invoice_5.id}")
+    # merch_1 = Merchant.create(name: "Shop Here")
```

Comment:
We discussed this on slack but worth mentioning that we commented out these seeds because they were interfering with the query Mallory was attempting to run.